src/index.css

```diff
@@ -109,3 +109,44 @@ code {
   transform: scale(0.9);
   transition: opacity 300ms, transform 300ms;
 }
+
+/* Enroll Tracks */
+
+.custom-ant-modal {
+  width: 100% !important;
+  padding: 0 2rem !important;
+}
+
+.tracks-card {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
+
+.ant-radio-group {
+  width: 100% !important;
+}
+
+.ant-radio-button-wrapper {
+  height: 100% !important;
+  width: 100% !important;
+  padding: 0 !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5838854)